README.md

```diff
@@ -246,6 +246,7 @@ and disables the auto-detection feature:
 
 - `helm:init` initializes Helm by downloading a specific version
 - `helm:dependency-build` resolves the chart dependencies
+- `helm:dependency-update` verifies that the required chart dependencies are present
```

Comment:
colleague of mine (Carter Cundiff) contributed this functionality and forgot to update the README here